apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -71,6 +73,15 @@ public function register_meta_field() {
 		register_meta( 'post', '_starter_page_template', $args );
 	}
 
+	/**
+	 * Register rest api endpoint for side-loading images.
+	 */
+	public function register_rest_api() {
+		require_once __DIR__ . '/class-wp-rest-sideload-image-controller.php';
+
+		( new WP_REST_Sideload_Image_Controller() )->register_routes();
```

Comment:
Just to confirm there is no global namespace qualifier here because the `WP_REST_Sideload_Image_Controller` class is now in the `A8C\FSE` namespace?

Reply:
Right, no qualifier needed because the current namespace also applies to `WP_REST_Sideload_Image_Controller`.